src/pages/Edit/atoms.ts

```diff
@@ -150,6 +170,26 @@ export const validationsAtom = atom<string[]>((get) => {
   ) {
     validations.push('End date should be later than start date');
   }
+  if (formData?.fileAddressList && formData.fileAddressList?.length > 0) {
+    if (formData?.fileAddressList?.length > 1000)
+      validations.push('Please limit whitelist address number less than 1000');
```

Comment:
These messages should have transactions.

Reply:
cannot use i18n in unfunctional component

Reply:
@UncleBill Any advice?